internal/cli/status.go

```diff
@@ -947,12 +947,12 @@ func (c *StatusCommand) FormatStatusReportComplete(
 		statusReportComplete = "? UNKNOWN"
 	}
 
-	t, err := ptypes.Timestamp(statusReport.GeneratedTime)
```

Comment:
`ptypes.Timestamp` is deprecated https://pkg.go.dev/github.com/golang/protobuf/ptypes#Timestamp